apps/sms/js/dialog.js

```diff
@@ -21,6 +21,7 @@
     id: 'localizableStringWithArgument',
     args: { n: count }
   },
+  classes: ['specific-class'], // optional
```

Comment:
I thought that "options" is about the buttons; but I wasn't really sure actually. Do you think it would be better ?